opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -227,7 +227,19 @@ namespace Opm {
 
         using namespace std::placeholders;
         const auto handler = std::bind( handleKW, _1, schedule, props, n_xyz );
+
+#ifdef _MSC_VER
+        for (auto& x : section)
+        {
+            auto keywords = handler(x);
+            for (auto& keyword : keywords)
+            {
+                this->keywords.push_back(keyword);
+            }
+        }
+#else
         this->keywords = fun::concat( fun::map( handler, section ) );
```

Comment:
> really? I agree that the NPM packaging issues have nothing to do with this, but the fact that this community felt like it was a good idea to introduce abstraction for two lines of code has quite a bit of relevance in the current context IMO. 

The for loop is an abstraction over "two lines of code". Why doesn't that apply?

> IMO they first increase the mental load because you have to get familiar with them. They are only beneficial when they are so common that you don't need to think about them anymore...

And there's only one way of getting there.

> yeah, and in all cases I need to look up in the implementation what it means exactly to e.g. concatenate a map and a vector.
```
auto m = fun::map( f, seq );
return std::vector< int > { m.begin(), m.end() }
```

> let's compromise by saying that in C++ it is in the eye of the beholder and that at least the "old c++ dogs" are not very familiar with it ;)

Sure. But an old dog could learn new tricks.